src/NUnitFramework/tests/Constraints/ToleranceTests.cs

```diff
@@ -3,124 +3,112 @@
 using System;
 using NUnit.Framework.Constraints;
 
-namespace NUnit.Framework.Tests.Constraints
+namespace NUnit.Framework.Tests.Constraints;
```

Comment:
I'm not in favour of changing to file scoped namespace as part of a normal PR.
We have always said that layout changes should be done as separate PRs.

I don't mind changing globally if that is what we want. I like consistency and if we move it should be a separate PR with a rule enforcing this for future developments.

Reply:
Here I actually don't agree with you.  The "Boy Scout Rule",  clean up as you go - leave the camp site in a better shape before you leave, is a better way, imho.  

Globally doing large scale code changes just for pretty-ness have been done the last couple of years, making it now very hard to see  who actually did a real functional change, git blame is now worthless. 

Small layout changes that comes as part of the suggestions should be ok to do, that be spaces, braces or similar small stuff.  File scoped namespaces is to me just a removal of braces. 

Also, we don't need to fix all the guidelines by ourselves, imho we can just "follow the flow", and accept the changes ind defaults that comes as time goes by.

Refactoring code however, should be done separately, because it just may have some other consequences.